tests/testtypes/consensus-spec-tests/types_basics.go

```diff
@@ -7,12 +7,25 @@ package consensus_spec_tests
 import "github.com/prysmaticlabs/go-bitfield"
 
 //go:generate go run -cover ../../../cmd/sszgen -type SingleFieldTestStruct -out gen_single_field_test_struct_ssz.go
+//go:generate go run -cover ../../../cmd/sszgen -type SmallTestStruct -out gen_small_test_struct_ssz.go
+//go:generate go run -cover ../../../cmd/sszgen -type FixedTestStruct -out gen_fixed_test_struct_ssz.go
```

Comment:
I think you forgot to regenerate the files after chaning the fields to the uintxx types. Pls do that too.

Reply:
Just run `go generate` and should do the trick

Reply:
❯ go generate
warning: GOCOVERDIR not set, no coverage data emitted
2024/07/26 21:55:23 Generating SSZ bindings for: [SingleFieldTestStruct]
/home/lavishq/ssz/tests/testtypes/consensus-spec-tests/gen_fixed_test_struct_ssz.go:14:2: T (type uint8) does not satisfy ssz.commonBitsLengths
/home/lavishq/ssz/tests/testtypes/consensus-spec-tests/gen_fixed_test_struct_ssz.go:15:2: T (type uint64) does not satisfy ssz.commonBitsLengths
/home/lavishq/ssz/tests/testtypes/consensus-spec-tests/gen_fixed_test_struct_ssz.go:16:2: T (type uint32) does not satisfy ssz.commonBitsLengths
/home/lavishq/ssz/tests/testtypes/consensus-spec-tests/gen_small_test_struct_ssz.go:14:2: T (type uint16) does not satisfy ssz.commonBitsLengths
/home/lavishq/ssz/tests/testtypes/consensus-spec-tests/gen_small_test_struct_ssz.go:15:2: T (type uint16) does not satisfy ssz.commonBitsLengths
package github.com/karalabe/ssz/tests/testtypes/consensus-spec-tests has errors
exit status 1
types_basics.go:9: running "go": exit status 1

this error

Reply:
Yeah, you'll have to delete the two files. The generator tries to load the package to get the type and there's a build error so it aborts.

Reply:
You probably generate the fields originally with a bad field, and then you left the two files there, so changing the fields always resulted in an error from the leftover old generated code.

Reply:
Ideally the generator would "ignore" that one file it's generating a replacement for, but there's no easy way to do that, so this is something I have to live with.

Reply:
after deleting those files and running `go generate` 
tests are proceeding :)

so i will commit and push that once

Reply:
Very nice :). Those aren't the tests mind you, just the types used by the tests, but nonetheless it's great :)